### ISSUE TITLE
Unable To Save Both Unsync and Sync Lyrics For Some Flac Files

### ISSUE THREAD

**Issue by nlogozzo** [Zeugma440/atldotnet#224]
Hello again :)

As the title suggests there seems to be some issues with flac files containing both unsync and sync lyrics. For some files, it works and both can be stored together no problem. For others, both unsync and sync lyrics cannot be stored together (usually just unsync lyrics are saved and the sync lyrics get cleared). 

Of course this is the code we are using to store lyrics is just:
```csharp
track = new Track(Path);
track.Lyrics.LanguageCode = "eng";
track.Lyrics.Description = "lyric";
track.Lyrics.UnsynchronizedLyrics = "test";
track.Lyrics.SynchronizedLyrics.Add(new LyricsInfo.LyricsPhrase(timestamp, "A"));
track.Save();
track = new Track(Path); //reload
```

Here's a link to files to test: https://drive.proton.me/urls/9X6MJ66ES4#qI3QPdyCT7qi
- `Annihilate.flac` works correctly with no issues...stores both unsync and sync lyrics
- `Numb.flac` is the one that doesn't work...the sync lyric doesn't get saved

**Comment by Zeugma440** [comment]
Hey there,

- **Annihilate** has its metadata inside an ID3v2 tag, which contains both synch `SYLT` and unsynch `USLT` lyrics ✅ 

- **Numb** has its metadata inside Vorbis tags. But the Vorbis tagging system doesn't have two different fields to store lyrics; it just has the `LYRICS` field for unsynched or LRC-formatted lyrics

=> We have two options there

1/ When writing to Vorbis or MP4/M4A tags, override unsynched lyrics when you set synched lyrics (which are richer by definition). They will get saved as LRC lyrics inside the dedicated field.

2/ Add a `Setting` to force ID3v2 tag creation when there's none inside the file

Any preference/suggestion?

**Comment by nlogozzo** [comment]
I guess nunber 1 would be best. Could we add a property to LyricsInfo maybe called `SupportsBothUnsyncAndSync` that we can set per tag type that way i can customize the UI based on whether a file supports both or not...

**Comment by Zeugma440** [comment]
> maybe called SupportsBothUnsyncAndSync that we can set per tag type that way i can customize the UI based on whether a file supports both or not...

Not sure that's where you want to go, knowing formats with a single lyrics field can very well support either of them (i.e. text block for unsynch or LRC for synch).

In terms of UI, I'd give the choice between Synched or Unsynched, but not give the option to edit both at the same time. Who wants to edit synched lyrics that contradict unsynched ones anyway?

**Comment by Zeugma440** [comment]
Available on today's v5.06
